winui/Rating/Customization.md

```diff
@@ -108,6 +108,30 @@ rating.ItemSize = 50;
 
 ![Size customization in WinUI Rating control](Rating_images/winui_rating_itemsize.png)
 
+## Orientation 
+
+The Rating control provides support to displayed horizontally or vertically by setting the `Orientation` property.
+
```

Comment:
@Priyanka-Vijayasankar The Rating control provides support to display the rating items either horizontally or vertically by setting the `Orientation` property. The default value is horizontal.

Reply:
Changed the content as mentioned above.